samples/bluetooth/central_iso/src/main.c

```diff
@@ -60,13 +62,13 @@ static void iso_timer_timeout(struct k_work *work)
 
 	net_buf_add_mem(buf, buf_data, len_to_send);
 
-	ret = bt_iso_chan_send(&iso_chan, buf);
+	ret = bt_iso_chan_send(&iso_chan, buf, sn++, BT_ISO_TIMESTAMP_NONE);
 
 	if (ret < 0) {
 		printk("Failed to send ISO data (%d)\n", ret);
 	}
 
-	k_work_schedule(&iso_send_work, K_MSEC(1000));
+	k_work_schedule(&iso_send_work, K_USEC(interval_us));
```

Comment:
Timing is changed with a factor of ten here - is that intentional?

Reply:
Timing was changed to match the SDU interval, which is intentional as we want to use this callback to increment the SN per SDU interval